fontbe/src/orchestration.rs

```diff
@@ -127,107 +114,47 @@ impl From<WorkId> for AnyWorkId {
     }
 }
 
-/// <https://learn.microsoft.com/en-us/typography/opentype/spec/glyf>
+/// A glyph and its associated name
 #[derive(Debug, Clone)]
-pub enum Glyph {
-    Simple(GlyphName, SimpleGlyph),
-    Composite(GlyphName, CompositeGlyph),
+pub struct NamedGlyph {
+    pub name: GlyphName,
+    pub glyph: Glyph,
 }
```

Comment:
NamedGlyph suggests the existence of other types but there really aren't any. I'd rather call this Glyph and rename the Glyph nested within, e.g.

```rust
/// A glyph and its associated name
///
/// See <https://learn.microsoft.com/en-us/typography/opentype/spec/glyf>
#[derive(Debug, Clone)]
pub struct Glyph {
    pub name: GlyphName,
    pub data: GlyphData,
}
```